src/python_testing/TC_SWTCH.py

```diff
@@ -835,6 +859,13 @@ def steps_TC_SWTCH_2_6(self):
                          "Verify that the TH receives MultiPressComplete event with PreviousPosition set to 1 and TotalNumberOfPressesCounted set to 1 from the DUT"),
                 ]
 
+    def pics_TC_SWTCH_2_6(self) -> list[str]:
```

Comment:
apparently we double-check for run_if_endpoint_matches for these tests and those cannot have pics (we check matching conditions, not pics here)

Reply:
That is then a bug...

Reply:
no, it's not - these tests are meant to be selected unconditionally, so they have a blank PICS (ie, they're not gated on any condition). TH selects these always, then the test reports if it is applicable.

Reply:
run if endpoint matches is just a slightly different mechanism for gating on the PICS.

Reply:
@cecille said it's not needed.